api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -33,6 +33,8 @@ service AccessListService {
   rpc GetAccessList(GetAccessListRequest) returns (AccessList);
   // UpsertAccessList creates or updates an access list resource.
   rpc UpsertAccessList(UpsertAccessListRequest) returns (AccessList);
+  // UpdateAccessList updates an access list resource.
+  rpc UpdateAccessList(UpdateAccessListRequest) returns (AccessList);
```

Comment:
What is the difference between update and upsert? Those two methods sound confusing when you want to modify an access list.

Reply:
Upsert creates or unconditionally updates the access list - i,e. it doesn't care about revision. It will just replace whatever is in backend.

Update cares about revision and forces the revision to match before doing the update. If the revision doesn't match, it returns an error `trace.CompareFailed`